_includes/assets-site-head.html

```diff
@@ -1,6 +1,6 @@
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/4.1.1/normalize.min.css">
 <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,600italic,700,700italic&subset=latin,latin-ext" type="text/css">
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css">
-<link rel="stylesheet" href="{{ '/css/styles.css?v=' | append: site.github.build_revision | relative_url }}">
+<link rel="stylesheet" href="/dest/css/libraries-main.css">
```

Comment:
Yes, this is a result of using these partials as both something you can grab and use in a project and to run this style guide. That libraries-main.css needs to be included in the head of any project  that wants to use the style guide, but is compiled in a different place for this standalone guide. I figured the most useful thing would be to put the link in the head partial so it's easy to grab even though the link will be broken when running the guide on it's own. Let me know what you think about it with that context...